plugins/themes/default/styles/objects/article_details.less

```diff
@@ -286,6 +287,19 @@
 
 		.item {
 			padding: @double;
+
+			.label {
+				margin: 0 0 @double;
+				font-family: @font-heading;
+				font-size: @font-bump;
+				font-weight: @bold;
+
+				&.doi .label,
+				&.keywords .label {
+					display: inline;
+					font-size: @font-base;
+				}
```

Comment:
These need to be placed up a level so that the `&.doi` attaches itself to the `.item` instead of the `.label`. So this:

```
.item {
  .label {
    &.doi .label {
       ..
    }
  }
}
```

Will produce a CSS selector like this:

```
.item .label.doi .label { ... }
```

But we actually want it to be:

```
.item.doi .label { ... }
```

So this block needs to come up a level, like this:

```
.item {
  ...
  
  .label {
    ...
  }

  &.doi .label,
  &.keywords .label {
    ...
  }
}
```

Reply:
This as well!